languagetool-language-modules/ga/src/main/resources/org/languagetool/rules/ga/grammar.xml

```diff
@@ -5411,6 +5411,22 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
             <short>Neamhchaighdeánach</short>
             <example correction="cá mhéad"><marker>cá mhéid</marker></example>
         </rule>
+        <rule id="AG_DEIREADH_AN_LAE" name="Ag deireadh an lae">
+            <antipattern>
+                <token>ag</token>
+                <token>deireadh</token>
+                <token>an</token>
+		<token>lae</token>
+            </antipattern>
```

Comment:
`<antipattern>` is used for exceptions to what would be matched by `<pattern>`; this cannot possibly be matched by the pattern below, so it's not needed